### ISSUE TITLE
Some members are in the wrong library

### ISSUE THREAD

**Issue by mansellan** [WaynePhillipsEA/twinbasic#647]
**Describe the bug**
Some members are in the wrong library

**To Reproduce**
```vb
    Dim x As Long
    x = VBRUN.KeyCodeConstants.vbKey0 ' Doesn't compile in twinBASIC, as it's in the VBA library not VBRUN
```

**Expected behavior**
Members in the same library as VBx

**Desktop (please complete the following information):**
 - OS: Windows 10
 - twinBASIC compiler version: v0.13.68

**Additional context**
Not checked but I suspect there's a few others

**Comment by WaynePhillipsEA** [comment]
Yes, everything is stuffed into VBA at the moment, I'll start splitting it up.  Thanks.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.13.69.  Thanks!
